src/engine/graph_node.h

```diff
@@ -140,15 +140,38 @@ struct DisplayListNode {
     struct DisplayListNode *next;
 };
 
+struct DisplayListHead {
```

Comment:
Nitpick: too similar of a name to `gDisplayListHead`

Reply:
Renamed to `DisplayListLinks`